msbuild/Xamarin.MacDev.Tasks.Core/Tasks/AlToolTaskBase.cs

```diff
@@ -82,5 +96,28 @@ string GetFileTypeValue ()
 				default: throw new NotSupportedException ($"Provided file type '{FileType}' is not supported by altool");
 			}
 		}
+
+		void LogErrorsFromOutput (string output)
+		{
+			try {
+				if (string.IsNullOrEmpty(output))
+					return;
+
+				var plist = PObject.FromString (output) as PDictionary;
+				var errors = PObject.Create (PObjectType.Array) as PArray;
+				var message = PObject.Create (PObjectType.String) as PString;
+
+				if ((plist?.TryGetValue ("product-errors", out errors) == true)) {
+					foreach (var error in errors) {
+						var dict = error as PDictionary;
+						if (dict?.TryGetValue ("message", out message) == true) {
+							Log.LogError (ToolName, null, null, null, 0, 0, 0, 0, "{0}", message.Value);
+						}
+					}
+				}
+			} catch (Exception ex) {
+				Log.LogWarning ("Failed to parse altool output: {0}", ex.Message);
```

Comment:
Should you include `output` in the exception log ?
That way you can get a clue of what went wrong

Reply:
also `$"Failed to parse altool output: {ex.Message}"` looks nicer :)